lib/src/main/java/io/ably/lib/realtime/AblyRealtime.java

```diff
@@ -71,6 +71,13 @@ public void onAuthUpdated() {
 	}
 
 	/**
+	 * Authentication error occurred
+	 */
+	public void onAuthError(ErrorInfo errorInfo) {
```

Comment:
Can't we make this package scope so it's not visible to users of the library (and `onAuthUpdated()`) ?